source/unix/openssl_rsa.c

```diff
@@ -11,7 +11,10 @@
 #include <openssl/evp.h>
 
 #if !defined(OPENSSL_IS_AWSLC) && !defined(OPENSSL_IS_BORINGSSL)
```

Comment:
nit: I think we should probably use defined(OPENSSL_IS_OPENSSL) instead. This check may break in the future if we add more variants.

Reply:
openssl does not define OPENSSL_IS_OPENSSL. we do define it manually in another place to the exact same thing as above.